### PR TITLE
App bundle updates

### PR DESCRIPTION
**Pull request by kgerheiser**
* Add `jedi-` prefix to `jedi-ufs` and `jedi-fv3`.
* Create a ufs-weather-model envrionment and bundle
* Create nceplibs bundle
* Updated Github Action to build multiple apps (https://github.com/kgerheiser/spack-stack/actions/runs/1816233302)

### CODE REVIEW

File: configs/repos/ufs/packages/ufs-weather-model-bundle-env/package.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+
+from spack import *
+
+class UfsWeatherModelBundleEnv(BundlePackage):
+    """Development environment for ufs-weathermodel-bundle"""
+
+    homepage = "https://github.com/JCSDA-internal/fv3-bundle"
+    git      = "https://github.com/JCSDA-internal/fv3-bundle.git"
+
+    maintainers = ['climbfuji', 'rhoneyager']
```
Comment (climbfuji) [inline configs/repos/ufs/packages/ufs-weather-model-bundle-env/package.py]: This should be you and possibly me and possibly someone from EPIC, not Ryan.

File: configs/repos/ufs/packages/ufs-weather-model-bundle-env/package.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+    version('main', branch='main')
+
+    depends_on('base-env', type=('build', 'run'))
+
+    depends_on('cmake', type=('build', 'run'))
```
Comment (climbfuji) [inline configs/repos/ufs/packages/ufs-weather-model-bundle-env/package.py:21]: List of packages needs updating.

**Review by climbfuji — APPROVED** [review]
I think it's a good first step, thanks. We'll need to make a few changes as noted here, but we can make them over the next fe days.

File: configs/repos/ufs/packages/ufs-weather-model-bundle-env/package.py (hunk 1 above)
Comment (kgerheiser) [reply configs/repos/ufs/packages/ufs-weather-model-bundle-env/package.py]: Ah, yes, Forgot to change the names when I copied.

**Review by climbfuji — APPROVED** [review]

**Pull request merged**